src/logcollector/read_mssql_log.c

```diff
@@ -133,7 +133,7 @@ void *read_mssql_log(logreader *lf, int *rc, int drop_it) {
             }
 
             /* Add additional message to the saved buffer */
-            if (sizeof(buffer) - buffer_len > str_len + 256) {
+            if (sizeof(buffer) - buffer_len > str_len) {
```

Comment:
Why was the `256` removed here? Is there a constant we could use instead?

Reply:
Because 256 is the value of OS_LOG_HEADER and I subtracted it in the buffer size.

Reply:
Got it, thanks!